dist/templates/ovnkube-node.yaml.j2

```diff
@@ -36,67 +36,6 @@ spec:
       {{ "hostPID: true" if ovn_unprivileged_mode=="no" }}
       containers:
 
-      # ovsdb-server and ovs-switchd daemons
```

Comment:
@aojea I think you will need to wrap the `ovs-deamons` container in this control statement
```
        {% if kind is defined and kind -%}

        {% endif %}
```
That way it will help someone who is trying out OVN K8s to get an hang of it having everything run as K8s Pods is simpler as they don't have to worry with any pre-installation of OVS on the host and such.

Reply:
hmm the CI uses kind and the goal is to not use  ovs as pods.
I've modified the kind.sh script to take care of that, meanwhile the people use the contrib/kind.sh it will work.

Reply:
I'm going to add a flag to add ovs-daemonset as an opt-in option,